### PR TITLE
POC to recover previous dagger state

### PR DESCRIPTION
**Pull request by pastuxso**
This update introduces support for persisting the latest Dagger output. This ensures that the Dagger component’s state remains intact when interacting with cell outputs, such as adjusting settings, and preventing any potential loss of data or state during such interactions.

Closes: #1490

### CODE REVIEW

File: examples/dagger/Notebook.md [hunk 1]
```diff
@@ -9,23 +9,22 @@ Showcase the notebook experience to **author, debug, express, and run** Dagger p
...
+
 ```
 
 ### What does the 🐮 cow say?
 
 ```sh {"id":"01J022WD7Z6TM1QQ075X09BTK4","interactive":"true","name":"COWSAY"}
 dagger --progress=plain \
     call \
-        -m github.com/shykes/daggerverse/wolfi@v0.1.2 \
+        -m github.com/shykes/daggerverse/wolfi@v0.1.4 \
```
Comment (pastuxso) [inline examples/dagger/Notebook.md:27]: The previous version is crashing for me.

**Comment by sourishkrout** [comment]
Let's be sure this edge case is solved for @pastuxso if you haven't yet:

https://github.com/user-attachments/assets/90a166fe-4d2f-40fd-927b-6e4edf7f1080

**Comment by pastuxso** [comment]
@pastuxso Thank you for the video. It wasn't covered yet. I added the support to address it.

File: examples/dagger/Notebook.md (hunk 1 above)
Comment (sourishkrout) [reply examples/dagger/Notebook.md:28]: makes sense, i think dagger releases sometimes break old versions 👍

**Review by sourishkrout — APPROVED** [review]
✅ LGTM

**Pull request merged**